packages/dialtone-vue2/components/tooltip/tooltip.vue

```diff
@@ -12,25 +12,19 @@
       @mouseenter="onEnterAnchor"
       @mouseleave="onLeaveAnchor"
       @keydown.esc="onLeaveAnchor"
-      @touchstart="onTouchStart"
     >
       <!-- @slot Slot for the anchor element -->
       <slot
         name="anchor"
       />
     </span>
-    <dt-lazy-show
+    <div
       :id="id"
       ref="content"
-      :show="isVisible"
```

Comment:
`isVisible` is a leftover should be removed.

Reply:
thanks